src/locales/en-US/translation.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Can we use JavaScript object to store all the translations. Something like below. Then we can access the key values easily I guess. @senuravihanjayadeva @miyurugunarathna @nisalrenuja please give your ideas as well.
```
{
   translation: {
      button: {
         cancel: "Cancel"
         submit: "Submit"
      }
   }
}
```